### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by ejurga**
Aaron I did some nice refactoring please merge.

### CODE REVIEW

File: wimhofbreather.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        self.script_dir = os.path.dirname(__file__)
+        if len(sys.argv) > 1 and sys.argv[1] == "debug":
+            self.testing_mode = True
+
+    def get_breathing_times(self):
+        return self.breathing_times
+
+    def lets_do_some_breathing(self):
+
+        for index in range(1, self.breathing_sets + 1):
```
Comment (op2rules) [inline wimhofbreather.py]: rename `index` to `iteration` or `round`

File: wimhofbreather.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        song = os.path.join(self.script_dir, f'audio/song{index}/' + random.choice(song_list))
+        music_player = vlc.MediaPlayer(song)
+
+        music_player.audio_set_volume(70)  # Wim doesn't talk very loud
+        music_player.play()
+
+        time_breathing = self.record_time_with_sentinel()
+        self.breathing_times.append(time_breathing)
+
+        if music_player.is_playing():
```
Comment (op2rules) [inline wimhofbreather.py:59]: Add comment

File: wimhofbreather.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            music_player.stop()
+
+    def play_holding_audio(self):
```
Comment (op2rules) [inline wimhofbreather.py]: rename `play_holding_audio` to `play_15_second_breath_hold`

File: wimhofbreather.py (hunk 1 above)
Comment (ejurga) [reply wimhofbreather.py]: done

File: wimhofbreather.py (hunk 2 above)
Comment (ejurga) [reply wimhofbreather.py:59]: okie

File: wimhofbreather.py (hunk 3 above)
Comment (ejurga) [reply wimhofbreather.py]: done, and comment removed

**Comment by ejurga** [comment]
All done!

**Pull request merged**